salt/modules/chroot.py

```diff
@@ -50,24 +50,25 @@ def __virtual__():
         return (False, 'Module chroot requires the command chroot')
 
 
-def exist(name):
+def exist(root):
     '''
```

Comment:
I would prevent changing the name of the already accepted `name` parameter in all these methods from the `chmod` module. Since this is not a major Salt update but these changes will be included as a patch in our 2019.2.0 package, we need to be careful here.

Whoever is already using the `chmod` module from our 2019.2.0 package will have troubles if they were using kwargs when calling those methods.

Reply:
I very much agree, but this module was never released yet. The code is in Neon, and the patch for neon with this change is pending.

Reply:
After doing some checks, I can confirm we neither release the yomi patch yet in any of our products, so it would be safe to merge this. Thanks! :+1: